Lib/glyphsLib/classes.py

```diff
@@ -141,6 +141,9 @@
 OFFCURVE = "offcurve"
 QCURVE = "qcurve"
 
+INSTANCETYPESINGLE = 0
+INSTANCETYPEVARIABLE = 1
```

Comment:
should this be defined in https://github.com/googlefonts/glyphsLib/blob/main/Lib/glyphsLib/builder/constants.py maybe? The same probably applies for the other constants defined in here, at least those that are imported by other modules and not private to the current one.

Reply:
Moving it there leads to circular imports. It's not immediately obvious to me how to avoid that ...